src/@chakra-ui/components/Tag/index.ts

```diff
@@ -112,10 +112,26 @@ const variantOutline = definePartsStyle((props) => {
   }
 })
 
+const variantHighContrast = definePartsStyle((props) => {
+  const { status = "normal" } = props
+  const defaultStyles = tagTheme.variants?.outline(props)
+  const statusStyles = getStatusStyles(status, "highContrast")
+  return {
+    container: {
+      ...defaultStyles?.container,
+      boxShadow: "none",
+
+      // Remove default dark mode styles
+      _dark: {},
+      ...statusStyles.container,
```

Comment:
Is this correct?

Light:
<img width="1310" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/e3b14ded-7304-4cab-ad87-f9c2cd670c5c">

Dark:
<img width="925" alt="image" src="https://github.com/ethereum/ethereum-org-website/assets/54227730/d4ac1586-86d4-412b-9d1d-9b523c9ac798">

For the background I see `body.light` in light mode, and `body.base` in dark

Reply:
![Screen Shot 2024-04-24 09 04 49 AM](https://github.com/ethereum/ethereum-org-website/assets/1120748/50fb0654-45c2-4768-bf37-78814ef88e88)

it should be body.light on both

Reply:
Hm... my comment doesn't seem to align with my screenshots as I look back at this... thanks, can disregard